packages/oft-evm/contracts/OFTCore.sol

```diff
@@ -174,7 +174,7 @@ abstract contract OFTCore is IOFT, OApp, OAppPreCrimeSimulator, OAppOptionsType3
         SendParam calldata _sendParam,
         MessagingFee calldata _fee,
         address _refundAddress
-    ) external payable virtual returns (MessagingReceipt memory msgReceipt, OFTReceipt memory oftReceipt) {
+    ) public payable virtual returns (MessagingReceipt memory msgReceipt, OFTReceipt memory oftReceipt) {
```

Comment:
Why was this made public?

Reply:
In order to call `super.send(...)` in `NativeOFTAdapter.send(...)` we had to change this from external to public. Do you foresee any issues with this change?

Reply:
This wont effect things down the line, or break interfaces etc. we should probably make this public imo

Reply:
It does make it slightly more expensive, but I see that it is needed.  I don't disagree.

Reply:
What about moving the body to `internal` function (`_send` or smth) and using that instead of modifying the interface?

Reply:
I like that idea because it creates a consistent pattern between `OFTCore.send` and `NativeOFTAdapter.send` that can be referenced in similar future implementations.

I have a PR here that does exactly that: https://github.com/LayerZero-Labs/devtools/pull/877